hlint.cabal

```diff
@@ -68,7 +68,7 @@ library
         refact >= 0.3,
         aeson >= 1.1.2.0,
         filepattern >= 0.1.1,
-        ghc-lib-parser-ex == 8.8.2
+        ghc-lib-parser-ex == 8.8.*
```

Comment:
Surely this must be 8.8.3.*, because you changed the API now?

Reply:
Good point. Should that read `8.8.3.*`?

Reply:
Yes

Reply:
Thanks for confirming. Will fix!

Reply:
>> Surely this must be 8.8.3.*

> Should that read 8.8.3.*?

😁 Sleepy...